packages/world/mud.config.ts

```diff
@@ -99,6 +99,26 @@ export default mudConfig({
       },
       tableIdArgument: true,
     },
+    KeysInTable: {
+      directory: "/modules/keysintable/tables",
+      primaryKeys: {},
+      schema: "bytes32[]", // For now only supports 1 key per value,
+      tableIdArgument: true,
+    },
+    KeysInTableLength: {
+      directory: "/modules/keysintable/tables",
+      primaryKeys: {},
+      schema: "uint32",
+      tableIdArgument: true,
+    },
```

Comment:
Why don't we combine these two to a single table with schema `{ keys: "bytes32[]", length: "uint32" }`?

Reply:
I was thinking that since `KeysInTableLength` is a workaround until there's a Store-native length getter, this way is easier to refactor

Reply:
Exactly. I have unified the tables for now anyway

Reply:
> I was thinking that since KeysInTableLength is a workaround until there's a Store-native length getter, this way is easier to refactor

Oh I see! That makes sense, but also fine with still unifying for now